testsuite/MDAnalysisTests/formats/test_libdcd.py

```diff
@@ -351,8 +352,10 @@ def write_dcd(in_name, out_name, remarks='testing', header=None):
             f_out.write(xyz=frame.xyz, box=frame.unitcell)
 
 
-@pytest.mark.xfail(os.name == 'nt' and sys.maxsize <= 2**32,
-                   reason="occasional fail on 32-bit windows")
+@pytest.mark.xfail((os.name == 'nt'
+                    and sys.maxsize <= 2**32) or
+                    platform.machine() == 'aarch64',
+                   reason="occasional fail on 32-bit windows and ARM")
```

Comment:
this doesn't actually fail on the `gcc 115` gcc compile farm ARM64 machine, but the test is clearly already flaky on some other platforms--maybe depends on character encoding/support stuff?